cache.go

```diff
@@ -0,0 +1,37 @@
+package gdrive
+
+import (
+	"sync"
+
+	. "github.com/dgraph-io/ristretto"
+)
+
+var (
+	cache *Cache
```

Comment:
Use global var makes it hard to test and maintain, how about add `newCache() *Cache` and keep it inside `Storage`?